### ISSUE TITLE
Enhancement: simpler link creation

### ISSUE THREAD

**Issue by rzw** [zadam/trilium#263]
When a URL is marked, pressing CTRL+K or click on link symbol should have the URL-field pre-filled with the marked URL.

**Comment by zadam** [comment]
You're right that current behavior is not ideal. In practice it's not a big deal to me since I usually paste links from clipboard and then I can just paste the link into the edit link dialog (after CTRL-K). I'll think about this though ...

**Comment by yozlet** [comment]
Many existing document editors & notes apps do this (e.g. Google Docs, GMail compose window), and it's super helpful.
